notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<p>It may be useful to point out that there are different "types" of flux columns, and link to the MAST documentation about them. Maybe here? https://heasarc.gsfc.nasa.gov/docs/tess/LightCurveFile-Object-Tutorial.html</p><p> </p><p>If we are looking for transits, do we want to use PDCSAP instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I think we still want to use SAP but I will let Thomas confirm, if thats okay.</p>

Reply:
<p>added text to point out the different fluxes</p><p> </p>

Reply:
<p>I think we are using the SAP because we are highlighting how to clean data using data quality flags. The PDCSAP has already gone through an initial round of cleaning. I think we should still use the SAP here. </p>

Reply:
<p>The relevant MAST resource is <a href="https://outerspace.stsci.edu/display/TESS/2.0+-+Data+Product+Overview" target="_blank">Data Product Overview</a>, from the TESS Archive Manual</p><p> </p><p>PDCSAP is "cleaner" than SAP (since long-term trends are removed). However, you should still be able to plot the "bad" data and then use flags to remove it</p>